src/main/kotlin/com/chattriggers/ctjs/minecraft/libs/renderer/Renderer.kt

```diff
@@ -760,11 +775,11 @@ object Renderer {
      * @param renderBlackBox render a pretty black border behind the text
      * @param scale the scale of the text
      * @param increase whether to scale the text up as the player moves away
+     * @param visibleThroughWalls whether to show the text through walls
      */
     // TODO(BREAKING): Rename from drawString (in Tessellator)
     // TODO(BREAKING): Remove color argument. Users should use colorize()
-    // TODO(CONVERT): Currently does not work in 1.17
-    // TODO(CONVERT): Is the scale argument necessary? Does scale() affect this function?
+    // TODO(BREAKING): Add visibleThroughWalls argument. Not sure if this is breaking or not if you leave it out.
```

Comment:
This shouldn't be breaking, right? since it has a default

Reply:
Checking this right now.

Yeah never mind, it doesn't break.